_inc/client/state/site/reducer.js

```diff
@@ -149,6 +178,15 @@ export function getAvailablePlans( state ) {
 	return get( state.jetpack.siteData, [ 'data', 'sitePlans' ] );
 }
 
+/**
+ * Returns all wpcom products
```

Comment:
Let's update this to say what this actually returns (I guess it won't return all WP.com products).